mathics/core/parser/parser.py

```diff
@@ -459,6 +459,20 @@ def p_Decrement(self, token):
         q = prefix_ops['PreDecrement']
         return Node('PreDecrement', self.parse_exp(q))
 
+    def p_PatternTest(self,token):
+        self.consume()
+        q = prefix_ops['Definition']
+        child = self.parse_exp(q)
+        return Node('Information', child, Node('Rule',Symbol("LongForm"), Symbol("False")))
```

Comment:
nitpick: some formatting (space after comma)

Reply:
I think all my changes now are mostly pep8 compliant, excepting for a few long lines.